--- a/src/wraplibxml.cpp	2026-03-07 01:21:32.681750347 +0100
+++ b/src/wraplibxml.cpp	2026-03-07 01:21:32.683286974 +0100
@@ -706,7 +706,7 @@ int WrapLibxml::saveEncoding (
 
 wxString WrapLibxml::getLastError()
 {
-	xmlErrorPtr err = xmlGetLastError();
+	const xmlError *err = xmlGetLastError();
 
 	if ( !err )
 		return nonParserError;
@@ -722,7 +722,7 @@ wxString WrapLibxml::getLastError()
 
 std::pair<int, int> WrapLibxml::getErrorPosition()
 {
-	xmlErrorPtr err = xmlGetLastError();
+	const xmlError *err = xmlGetLastError();
 	if ( !err )
 		return std::make_pair ( 1, 1 );
 
